starknet_py/tests/e2e/account/account_test.py

```diff
@@ -646,3 +647,68 @@ async def test_account_execute_v3(account, deployed_balance_contract):
         call=get_balance_call
     )
     assert initial_balance + 100 == balance_after_increase
+
+
+@pytest.mark.asyncio
+async def test_invoke_with_tip(account, hello_starknet_class_hash):
```

Comment:
nit: We always put version in the name:
```suggestion
async def test_invoke_with_tip(account, hello_starknet_class_hash):
```